committeeoversight/urls.py

```diff
@@ -19,5 +19,5 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'', include('committeeoversightapp.urls')),
+    url(r'', include('committeeoversightapp.urls'))
```

Comment:
In general I like to keep trailing commas in lists - otherwise, you risk forgetting to add one, when you put another element in your list.

Reply:
👍🏻